extensions/ext/GL_EXT_subgroupuniform_qualifier.txt

```diff
@@ -0,0 +1,164 @@
+Name
+
+    EXT_subgroupuniform_qualifier
+
+Name Strings
+
+    GL_EXT_subgroupuniform_qualifier
+
+Contact
+
+    Neil Henning (neil.henning 'at' amd.com)
+
+Contributors
+
+    Jeff Bolz, NVIDIA Corporation
+    John Kessenich, Google
+    Nicolai Hähnle, AMD
+
+Status
+
+    Draft
+
+Version
+
+    Last Modified Date:         September 19, 2018
+    Revision:                   1
+
+Number
+
+    TBD
+
+Dependencies
+
+    This extension requires GL_KHR_vulkan_glsl
+
+Overview
+
+    This extension adds a "subgroupuniformEXT" type qualifier and constructor,
+    such that for each dynamic instance of an object qualified by a subgroup
+    uniform type or constructed with a subgroup uniform constructor, all active
+    invocations in a subgroup compute the same result value.
+
+New Procedures and Functions
+
+    None.
+
+New Tokens
+
+    None.
+
+Modifications to GL_KHR_vulkan_glsl
+
+    Add to the "Mapping to SPIR-V" section
+    
+    Mapping of subgroupuniformEXT type qualifier:
+
+      subgroupuniformEXT -> subgroupuniformEXT decoration on variables
```

Comment:
Is that the right mapping? It is not to Uniform? At least the case seems incorrect, but I didn't see where the SPIR-V subgroupuniformEXT was introduced.

Reply:
Ah no doh, this was meant to be the Uniform decoration in SPIR-V but I copy+pasta fail. Lemme fix that quick.